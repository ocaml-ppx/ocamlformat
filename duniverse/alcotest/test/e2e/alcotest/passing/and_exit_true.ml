let () =
  let open Alcotest in
  let id () = () in
  run ~and_exit:true __FILE__
    [ ("test-a", [ test_case "Test case" `Quick id ]) ];
  Printf.printf "\n Should never be printed!%!";
  assert false
