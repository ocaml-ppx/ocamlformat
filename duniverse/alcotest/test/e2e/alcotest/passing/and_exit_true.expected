Testing `test/e2e/alcotest/passing/and_exit_true.ml'.
This run has ID `<uuid>'.

  [OK]          test-a          0   Test case.

Full test results in `<build-context>/_build/_tests/<test-dir>'.
Test Successful in <test-duration>s. 1 test run.
